Disables testing of FFS and UFS2 images (which always fail).  Upstream 
maintainers say that these can be ignored anyways.

http://lists.gnu.org/archive/html/bug-grub/2004-05/msg00076.html
http://bugs.gentoo.org/71811

--- grub-0.96/stage2/size_test
+++ grub-0.96/stage2/size_test
@@ -36,9 +36,9 @@
 }
 
 # The bootloader area of a FFS partition is 14 sectors.
-check ffs_stage1_5 7168
-
-check ufs2_stage1_5 7168
+#check ffs_stage1_5 7168
+#
+#check ufs2_stage1_5 7168
 
 # Stage 1.5 can be installed in the sectors immediately after MBR in the
 # first cylinder, so the size is (63 - 1) sectors.
